src/main/java/com/conveyal/datatools/manager/auth/Auth0Connection.java

```diff
@@ -188,4 +193,12 @@ public static void checkGTFSPrivileges(Request request) {
             }
         }
     }
+
+    /**
+     * This is used to activate behavior within this class that emulates a testing environment. Obviously, this is used
+     * and should only be used while running tests of this project.
+     */
+    public static void setInTestingEnvironment (boolean inTestingEnvironment) {
```

Comment:
Instead of using this method and the inTestingEnvironment  boolean, I think it might be cleaner to stash the testing url (localhost:8089) in a public constant in the test class and use a `isTestEnvironment` method that checks if the base URL equals the test URL. Really any changes we can make to reduce the number of times string literals are redefined or having to call multiple functions to do the same thing.

Reply:
I'm don't understand what you've got in mind here. It seems like this would mean bringing back [this code](https://github.com/conveyal/datatools-server/pull/187#discussion_r261688311) you wanted removed.

Reply:
Sorry for the confusion, I'll create a PR.